Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/User/AdditionalUserFieldsWindow.tsx

```diff
@@ -334,6 +334,6 @@ function ValueDisplay<V extends Application.Types.iAdditionalUserFieldValue>(pro
 		return <CheckBox<V> Record={props.Value} Field={'Value'} Label={''} Setter={props.Setter} />
 	else
 		return <Select<V> EmptyOption={true} Record={props.Value} Field={'Value'} Label={''} Setter={props.Setter}
-			Options={props.ValueListItems.map(x => ({ Value: x.ID.toString(), Label: x.Value }))} />
+			Options={props.ValueListItems.map(x => ({ Value: x.Value, Label: x?.AltValue ?? x.Value }))} />
```

Comment:
```suggestion
			Options={props.ValueListItems.map(x => ({ Value: x.Value, Label: x.AltValue ?? x.Value }))} />
``` 
either that or add an alternative because if `x` is null this will still fail

Reply:
That's a good point that the ? isn't really necessary. Will make the changes.